pkg/client/grpc/client.go

```diff
@@ -69,14 +72,25 @@ func buildDialOptions(o Options) []grpc.DialOption {
 	return res
 }
 
-func buildAuthInterceptor(o Options) grpc.UnaryClientInterceptor {
+func buildInterceptor(o Options) grpc.UnaryClientInterceptor {
```

Comment:
This should be two different interceptors. Think about it like middlewares.

Reply:
I tired, but the second one is override the first.